src/Components/PostCard/PostCard.js

```diff
@@ -0,0 +1,23 @@
+import CardArticle from '../UI/PostArticle.Styled';
+import CardAddress from '../UI/PostAdresse.Styled';
+import CardUserName from '../UI/PostName.Styled';
+
+export default function PostCard({ name, content, contact, mail, mobile }) {
+	return (
+		<CardArticle>
+			<CardUserName>{name}</CardUserName>
+
+			<p>{content}</p>
+
+			<p>{contact}</p>
```

Comment:
Is the value of `contact` always 'Kontakt' and will never be changed? If yes, you could generate the headline statically.

```suggestion
			<p>Kontakt</p>
```

Reply:
I have changed it now because I think it will always be 'Kontakt'.